src/datamodel_code_generator/__init__.py

```
@@ -130,22 +124,38 @@ def get_version() -> str:
 DEFAULT_MAX_VARIABLE_LENGTH: int = 100
 
 
+_pysnooper_initialized: bool = False
+
+
 def snooper_to_methods() -> Callable[..., Any]:
     """Class decorator to add pysnooper tracing to all methods."""
 
     def inner(cls: type[T]) -> type[T]:
-        if not pysnooper:
+        global _pysnooper_initialized  # noqa: PLW0603
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `PLW0603`)

Remove unused `noqa` directive

(RUF100)